lib/openstack_api/openstack_service.py

```diff
@@ -0,0 +1,50 @@
+from typing import Optional
+from openstack.connection import Connection
+from openstack.compute.v2.service import Service
+
+
+def disable_service(
+    conn: Connection,
+    service: Service,
+    hypervisor_name: str,
+    service_binary: str,
+    disabled_reason: str,
+) -> Optional[Service]:
+    """
+    Disables an Openstack service
+    :param conn: Openstack connection
+    :param service: The instance of the service class.
+    :param hypervisor_name: (str): The name or ID of the hypervisor.
```

Comment:
We don't need this as the type is in the parameters above
```suggestion
    :param hypervisor_name: The name or ID of the hypervisor.
```

Reply:
Oh yep nice, good spot, I think that must have been a copy and paste mistake.